### ISSUE TITLE
Cannot select lines with mouse from screen edge on KDE

### ISSUE THREAD

**Issue by jhard** [gnome-terminator/terminator#866]
When terminator is maximized (also in fullscreen-mode), I cannot start selecting lines when the mouse cursor is at the left screen edge.

Moving into the window by the tiniest amount makes it work as expected.
Double-Clicking while at the screen edge will also not select the line the mouse is hovering over.
Visually, the window (and the content area) extends to the screen edge.

I've unsuccessfully tested setting the window to borderless or fiddling with terminator options (Window Geometry Hints, Extra Styling, Window Borders etc).
Not maximizing the window and moving it beyond the edge of the screen makes it work, so I don't believe it's because of the mouse-location/screen-edge.

It happens with the default config as well.

I'm using terminator 2.1.1 on kubuntu 22.04 with Plasma 5.24.7. It does work in konsole, so it doesn't seem to be a general issue.

Is this something that is controlled by an option? Anything I can do to find out what's causing it (or make it easier for others to reproduce)?

**Comment by mattrose** [comment]
Have you tried running 2.1.3.  The package is available in my PPA (see [INSTALL.md](https://github.com/gnome-terminator/terminator/blob/51e3bd251952a14737e5cd7470e6e3ea3ba5ac23/INSTALL.md) )

**Comment by mattrose** [comment]
or you could try duplicating it with master as well.

**Comment by jhard** [comment]
Hey, thanks for the hint. I hadn't, but I've just installed it from the PPA. I've also checked out master and ran it from there.

Unfortunately, neither changed anything, I still need to move inwards ever so slightly when trying to select text from the left.

If there's anything useful I could provide or test (I know barely any python but if you have any approximate location in mind that might be responsible, I'm happy to poke around and change things), please let me know.

It's not the biggest of annoyances and I've somewhat gotten used to it, but it still catches me occasionally, so I'm happy to experiment a bit to help solve it.

**Comment by mattrose** [comment]
Try adding this to `~/.config/gtk-3.0/gtk.css`
```
vte-terminal {
    padding: 1px 1px 1px 10px;
}
```

That will give you a 10pixel padding on the left side of the screen.

**Comment by jhard** [comment]
Thanks, that set me on the way to solve it.
The padding doesn't change the behavior and I still can't select on the screen edge -- which I want because it's just the natural border to push the mouse to and start selecting.
I then fiddled around with the Gtk Inspector but could not get it to work by setting various properties either.

However, the ability to add padding gave me the idea to use KWin and resize + move the window ever so slightly, and that works. I've added a few px of padding on the left, resized the window to my monitor's width + 2px and then forced position on the left side to -2px (-1 did not have the intended effect). I can now select reliably from the left screen edge.

It has the slight drawback that I enforce the window to be maximized, but I'm easily distracted so I have it maximized 98% of the time anyhow, and I can work around the remaining cases. 

Again, thank you, both for your work on terminator and helping me make this annoyance go away (and making me learn how easy it is to inspect & manipulate Gtk applications' styling in the process).
